ui/pages/reservation-unit/[id].tsx

```diff
@@ -700,7 +719,13 @@ const ReservationUnit = ({
         reservationUnitPks: [reservationUnit.pk],
       };
 
-      setReservation({ begin, end, pk: reservationUnit.pk, price: null });
+      setReservation({
+        begin,
+        end,
+        pk: reservationUnit.pk,
+        price: null,
+        reservationUnitPk: reservationUnit.pk,
```

Comment:
Why is the pk === reservationUnitPk

Reply:
Good catch, was some relic that had no effect on anything.